scripts/migrations/007-experimental-false.test.js

```diff
@@ -0,0 +1,69 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const assert = require('assert').strict;
+
+const { fixExperimental } = require('./007-experimental-false.js');
+
+describe('fixExperimental()', () => {
+  it('chrome + firefox + safari', () => {
+    const bcd = {
+      api: {
+        fetch: {
+          __compat: {
+            support: {
+              chrome: {
+                version_added: '100',
+              },
+              firefox: {
+                version_added: '100',
+              },
+              safari: {
+                version_added: '15',
+              },
+            },
+            status: {
+              experimental: true,
+              standard_track: true,
+              deprecated: false,
+            },
+          },
+        },
+      },
+    };
+    assert.equal(bcd.api.fetch.__compat.status.experimental, true);
+    fixExperimental(bcd);
+    assert.equal(bcd.api.fetch.__compat.status.experimental, false);
+  });
+
+  it('just chrome + firefox', () => {
+    const bcd = {
+      api: {
+        fetch: {
+          __compat: {
+            support: {
+              chrome: {
+                version_added: '100',
+              },
+              firefox: {
+                version_added: '100',
+              },
+              safari: {
+                version_added: false,
+              },
+            },
+            status: {
+              experimental: true,
+              standard_track: true,
+              deprecated: false,
+            },
+          },
+        },
+      },
+    };
+    assert.equal(bcd.api.fetch.__compat.status.experimental, true);
+    fixExperimental(bcd);
+    assert.equal(bcd.api.fetch.__compat.status.experimental, true);
```

Comment:
Shouldn't experimental be set to `false`, since the feature is supported in two different browser engines?

Reply:
I have a follow-up to count engines instead, but I'd like to update and use this script in multiple phases, to start with the really simple cases. More scrutiny in review will be needed for the "two different engines" changes I think.